common/changes/office-ui-fabric-react/focus2_2018-03-09-17-45.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Removed focusability on buttons for split buttons and spin buttons",
```

Comment:
Can you please format comments here as:

<component>: Change that was made, which shows up in release notes. (this can be markdown)

E.g.
Split buttons/SpinButton: Adjusted alignment to previous left alignment.

Also! this comment you have provided seems very wrong! you removed focusability on buttons???

Reply:
Oops, you are absolutely correct, this change file was for something else that shouldn't have been merged in. Sorry about that, I will update it with the appropriate change list.